etl/grapher_helpers.py

```diff
@@ -179,6 +181,13 @@ def country_to_entity_id(
         return cast(pd.Series, entity_id.astype(int))
 
 
+def get_or_create_entity(name: str) -> int:
+    cursor = get_connection().cursor()
+    db = DBUtils(cursor)
+    print(name)
+    return db.get_or_create_entity(name)
```

Comment:
Not anything you need to change now, but as a comment it seems like we're only ever creating entities here, never getting them. Getting them would mean that our countries/regions is out of sync with the grapher country data.